nixosTests/tests/hermes-test.nix

```diff
@@ -116,6 +126,7 @@ in
 
         services.hermes = {
           enable = true;
+          global.log_level = "trace";
```

Comment:
I would use `debug` here by default as `trace` is very very noisy and not that helpful for relaying in general.

Reply:
Oh sorry just saw that the actual default is `info`, and it's only trace for the test. Nevermind.